FLATBUFFERS_VTABLE_UNDERLYING_TYPE should be set in flatbuffers/base.h but for
some reason it is not. It's not needed, so we create an empty define, otherwise
schema_generated.h will throw a lot of errors.

diff --git a/tensorflow/lite/schema/schema_generated.h b/tensorflow/lite/schema/schema_generated.h
index d884d7f8..17fdca54 100755
--- a/tensorflow/lite/schema/schema_generated.h
+++ b/tensorflow/lite/schema/schema_generated.h
@@ -20,6 +20,10 @@ limitations under the License.

 #include "flatbuffers/flatbuffers.h"

+#ifndef FLATBUFFERS_VTABLE_UNDERLYING_TYPE
+#define FLATBUFFERS_VTABLE_UNDERLYING_TYPE
+#endif
+
 namespace tflite {

 struct CustomQuantization;
